templates/main-nav.html

```diff
@@ -40,8 +40,13 @@
       <ul class="hover-menu">
         <li><a href="/topics/design/">Design</a></li>
         <li><a href="/topics/juju/">Juju</a></li>
+        <li><a href="/topics/kernel/">Kernel</a></li>
+        <li><a href="/topics/lxd/">LXD</a></li>
         <li><a href="/topics/maas/">MAAS</a></li>
+        <li><a href="/topics/security/">Security</a></li>
         <li><a href="/topics/snappy/">Snappy</a></li>
+        <li><a href="/topics/desktop/">Ubuntu desktop</a></li>
+        <li><a href="/topics/server/">Ubuntu Server</a></li>
```

Comment:
Why do these need to be prefixed with `Ubuntu`? I think its clear they are Ubuntu.